### PR TITLE
RTP4

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RealtimeClientPresence.swift [hunk 1]
```diff
@@ -77,6 +77,35 @@ class RealtimeClientPresence: QuickSpec {
...
+                let options = AblyTests.commonAppSetup()
+                var clientSource: ARTRealtime!
+                defer { clientSource.close() }
+
+                let clientTarget = ARTRealtime(options: options)
+                defer { clientTarget.close() }
+                let channel = clientTarget.channels.get("test")
+
+                channel.presence.subscribe { member in
+                    expect(member.action).to(equal(ARTPresenceAction.Enter))
```
Comment (tcard) [inline Spec/RealtimeClientPresence.swift]: This checks for ENTER, but the spec says to expect PRESENT events, I assume through SYNC messages.

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: Done 799aca5. Thanks.

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: Test is failing 😫. The client is broken. I will change to pending after rebasing with master.

**Comment by tcard** [comment]
LGTM

**Pull request merged**